x/consensus/keeper/keeper.go

```diff
@@ -14,15 +14,15 @@ import (
 var _ exported.ConsensusParamSetter = (*Keeper)(nil)
 
 type Keeper struct {
-	storeKey storetypes.StoreKey
+	storeSvc storetypes.KVStoreService
```

Comment:
two cents: maybe it is because I am used to it, but personally, I prefer `storeService` over `storeSvc`.

Reply:
ditto

Reply:
Will fix it, thank you